src/pages/chainlink-functions/resources/service-limits.md

```diff
@@ -6,20 +6,20 @@ title: "Chainlink Functions Service Limits"
+| ------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| Supported languages                                           | Supported language of the Source code that is submitted in the Chainlink Functions request                     | Only JavaScript is supported for the moment. Your source code can only use vanilla Node.js features (you cannot import modules) |
+| Maximum requests in flight                                    | The maximum number of simultaneous Chainlink Functions requests that are not fulfilled yet by the DON          | Limited by the [effective balance](/chainlink-functions/resources/concepts#subscriptions)                                       |
+| Maximum callback gas limit                                    | The maximum amount of gas that you can set for Chainlink Functions to fulfill your callback function           | 300,000                                                                                                                         |
+| Maximum subscriptions                                         | Maximum subscriptions that you can create                                                                      | unbounded                                                                                                                       |
+| Maximum consumer contracts per subscription                   | The maximum number of consumer contracts that you can add to a subscription                                    | 100                                                                                                                             |
+| Request fulfillment timeout                                   | The maximum duration of an in-flight Chainlink Functions request. After this duration, the request is canceled | 5 minutes                                                                                                                       |
+| Maximum request (combined source code, args and secrets) size | The maximum size of a Chainlink Request. This includes the source code, arguments, and secrets                 | 30 kilobytes                                                                                                                    |
+| Maximum returned value size                                   | The maximum size of the value returned by the DON when fulfilling a request                                    | 256 bytes                                                                                                                       |
```

Comment:
Maybe say that it's "The maximum size of the value that your Function can return, in bytes."